src/main/cpp/VisionIO.cpp

```diff
@@ -0,0 +1,32 @@
+#include "io/VisionIO.h"
+void VisionIO::Periodic() {
+VOKC_CALL(ProcessIO());
+}
+void VisionIO::SimulationPeriodic() {
+
+}
+
+bool VisionIO::ProcessIO() {
+if (sw_interface_->update_config) {
+UpdateVisionConfig(sw_interface_->vision_config);
+sw_interface_->update_config = false;
+}
+ProcessInputs();
+SetOutputs();
+return true;
```

Comment:
Fix indentation (add tabs to each level of brackets). For example:
```
bool VisionIO::ProcessIO() {
(tab) if (...) {
(tab) (tab) code inside if
(tab) }
```
please do this style for the whole file.